docs/reference/driver-compatibility.txt

```diff
@@ -365,6 +367,7 @@ for that Ruby version is deprecated.
      - |checkmark|
      - |checkmark|
      - |checkmark|
+     - |checkmark|
```

Comment:
2.17-stable is not tested against 3.1, if this change is merged into master it needs to be backported there and the configurations added.